platform/security/rest/security-rest-cxfwrapper/src/test/java/org/codice/ddf/cxf/oauth/OAuthSecurityImplTest.java

```diff
@@ -232,6 +230,144 @@ public void testSettingSystemTokensOnClientErrorResponse() {
     assertTrue(form.asMap().get("grant_type").contains("client_credentials"));
   }
 
+  @Test
+  public void testUserTokensOnClientPasswordFlow() throws Exception {
```

Comment:
❓ Is this missing a test case where we use `grant_type=password` although we do not pass a `totp` in the additionalMap ? Or is that unnecessarily ? 

Reason:  We will be passing a totp when we are in testing mode. Although, in live mode, there will not be a totp.

Reply:
I don't think that matters I just used the totp as an example of a parameter that can be passed in for this unit test